src/util/framerate.ts

```diff
@@ -183,8 +183,11 @@ export class DownsamplingBasedOnFrameRateCalculator {
   constructor(
     public numberOfStoredFrameDeltas: number = 10,
     private maxDownsamplingFactor: number = 8,
-    private desiredFrameTimingMs = 1000 / 60,
+    private desiredFrameTimingMs = 1000 / 30,
     private downsamplingPersistenceDurationInFrames = 15,
+    /** If true, logs factor changes and timing stats to console */
```

Comment:
Could maybe make this an `@param` tag for documenting this for consistency with the other params in this constructor

Reply:
Alternatively neuroglancer already uses a number of DEBUG_XYZ consts for these kind of debugging logs which we could considering using instead. Not sure if @jbms has a guideline or preference there for this

Reply:
sounds good.. i think the DEBUG_XYZ approach is probably better. Will implement.